src/features/SendAssets/SendAssets.tsx

```diff
@@ -39,7 +41,7 @@ function SendAssets() {
       component: IS_ACTIVE_FEATURE.PROTECT_TX ? SendAssetsFormWithProtectTx : SendAssetsForm,
       props: (({ txConfig }) => ({ txConfig }))(txFactoryState),
       actions: (payload: IFormikFields, cb: any) => {
-        if (getProTxValue(['state', 'protectTxEnabled'])) {
+        if (getProTxValue(['state', 'protectTxEnabled']) && !isMyCryptoMember) {
```

Comment:
How about we change this syntax and access the values directly from the context?
```
const { PTxEnabled } = useContext(PTxContext)
```